src/backend/core/warren/conf.py

```diff
@@ -32,8 +33,10 @@ class Settings(BaseSettings):
     # API configuration
     MAX_DATETIMERANGE_SPAN: timedelta = timedelta(days=365)  # 1 year shift from since
     DEFAULT_DATETIMERANGE_SPAN: timedelta = timedelta(days=7)  # 7 days shift from until
+    VIDEO_PLAYED_THRESHOLD_DURATION: int = 30  # seconds
+    DATE_DAY_FORMAT: str = "YYYY-MM-DD"
```

Comment:
I think we could use the Django convention, e.g.:

```suggestion
    DATE_FORMAT: str = "YYYY-MM-DD"
```

WDYT?

Reply:
sounds good to me !